packages/app/src/app/components/CreateSandbox/Import/SuggestedRepositories.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import formatDistanceStrict from 'date-fns/formatDistanceStrict';
+import { zonedTimeToUtc } from 'date-fns-tz';
+import styled from 'styled-components';
+import VisuallyHidden from '@reach/visually-hidden';
+
+import {
+  Stack,
+  Text,
+  Icon,
+  InteractiveOverlay,
+  Button,
+  Element,
+} from '@codesandbox/components';
+import { v2DefaultBranchUrl } from '@codesandbox/common/lib/utils/url-generator';
+
+import { useActions, useAppState } from 'app/overmind';
+import { useGitHuPermissions } from 'app/hooks/useGitHubPermissions';
+
+import { fuzzyMatchGithubToCsb } from './utils';
+import { useGithubAccounts } from './useGithubOrganizations';
+import { useGitHubAccountRepositories } from './useGitHubAccountRepositories';
+import { StyledSelect } from '../elements';
+
+export const SuggestedRepositories = () => {
+  const { activeTeamInfo } = useAppState();
+  const { signInGithubClicked } = useActions();
+  const { restrictsPrivateRepos } = useGitHuPermissions();
+
+  const [selectedAccount, setSelectedAccount] = useState<string | undefined>();
+  const githubAccounts = useGithubAccounts();
+
+  const selectOptions = useMemo(
+    () =>
+      githubAccounts.data
+        ? [githubAccounts.data.personal, ...githubAccounts.data.organizations]
+        : undefined,
+    [githubAccounts.data]
+  );
+
+  useEffect(() => {
+    // Set initially selected account bazed on fuzzy matching
+    if (githubAccounts.state === 'ready' && selectedAccount === undefined) {
+      const match = fuzzyMatchGithubToCsb(activeTeamInfo?.name, selectOptions);
+
+      setSelectedAccount(match.login);
+    }
+  }, [githubAccounts.state, selectedAccount, activeTeamInfo, selectOptions]);
+
+  // eslint-disable-next-line no-nested-ternary
+  const selectedAccountType = selectedAccount
+    ? selectedAccount === githubAccounts?.data?.personal?.login
+      ? 'personal'
+      : 'organization'
+    : undefined;
```

Comment:
I think this is more readable than a `let` with a bunch of `if` and `else` statements.